modules/ROOT/pages/relnotes-go-sdk.adoc

```diff
@@ -0,0 +1,396 @@
+= Couchbase Go SDK Release Notes
+
+== SDK Installation
+
+Installing the Couchbase Go SDK is similar to installing most other Go projects:
+
+[source,bash]
+----
+$ go get gopkg.in/couchbase/gocb.v1
+----
+
+=== API Documentation
+
+The most current and up to date API Documentation is always available through the http://godoc.org/gopkg.in/couchbase/gocb.v1[godoc website].
+
+*Any checkouts of the Go SDK between July 10 2017 and Sept 29 2017 may have compatibility issues with Server 5.0.*
+
+== Version 1.4.1 (21 August 2018)
```

Comment:
We've had a 1.4.2 since this, which there are definitely release notes for. They show on [https://developer.couchbase.com/server/other-products/release-notes-archives/go-sdk](https://developer.couchbase.com/server/other-products/release-notes-archives/go-sdk)

Reply:
Thanks Charles - I had a note to add these back in.
Sadly too hurried to fit an extra task in at the end of the day, and missed it. :-(